conf/mini_provisioner/v2/ha_setup.yaml

```diff
@@ -18,6 +18,10 @@ ha:
         cmd: ha_setup post_install
         args: --config $URL
 
+    prepare:

```

Comment:
can you remove ha_setup.yaml 
covering in https://github.com/Seagate/cortx-ha/pull/373/files

Reply:
sure

Reply:
done @ajay-paratmandali